src/sentry_plugins/splunk/plugin.py

```diff
@@ -308,3 +308,4 @@ def post_process(self, event, **kwargs):
                 "event_type": event.get_event_type(),
             },
         )
+        tsdb.incr(tsdb.models.project_total_forwarded, event.project.id, count=1)
```

Comment:
@NisanthanNanthakumar I am wondering if it would be better to implement the `forward_event` method like we do for SQS and Segment so we don't have to override the `post_process` method in the base class. As it is, we are having to duplicate the logic for rate-limiting which isn't ideal. Cc @manuzope